src/main/java/com/deavensoft/paketomat/mapper/CitiesMapper.java

```diff
@@ -0,0 +1,18 @@
+package com.deavensoft.paketomat.mapper;
+
+import com.deavensoft.paketomat.center.model.City;
+import com.deavensoft.paketomat.city.dto.CitiesDto;
+import com.deavensoft.paketomat.city.dto.CityDto;
+import org.mapstruct.Mapper;
+import org.mapstruct.factory.Mappers;
+
+@Mapper(componentModel = "spring", uses = CityMapper.class)
+public interface CitiesMapper {
```

Comment:
The CitiesMapper is not needed. Please, remove it. 

Keep in mind, that the similar name of the classes mustn't confuse you. Class _Cities_ is not the same as _City_.  In the interface, you mixed _Cities_ and _City_ objects. 

- [ ] Remove interface

Reply:
- [ ] Delete the interface

Reply:
The interface is deleted.